docs/app/img/icons/sets/symbol-icons.svg

```diff
@@ -0,0 +1,4 @@
+<svg>
+<symbol id="spinner" viewBox="0 0 44 44"><path d="M5.386 19.817c-.584 4.44.595 8.84 3.32 12.39 5.628 7.333 16.172 8.722 23.504 3.095 7.332-5.628 8.72-16.173 3.093-23.504-3.57-4.65-9.406-7.12-15.238-6.447-.264.028-.51-.16-.54-.428-.03-.267.16-.51.43-.54 6.166-.707 12.345 1.902 16.12 6.822 5.956 7.76 4.488 18.917-3.27 24.87-3.215 2.467-7.01 3.66-10.78 3.66-5.33 0-10.604-2.385-14.092-6.932C5.048 29.042 3.8 24.386 4.42 19.69c.595-4.532 2.86-8.578 6.392-11.447H5.57c-.27 0-.488-.22-.488-.488 0-.27.218-.488.487-.488h6.678c.27 0 .488.218.488.488v6.68c0 .27-.218.487-.488.487s-.488-.217-.488-.488V8.737c-3.53 2.724-5.792 6.657-6.374 11.08z"><animateTransform attributeName="transform" type="rotate" from="0 22 22" to="360 22 22" dur="3s" repeatCount="indefinite"/></path></symbol>
```

Comment:
I like the AngularJS logo SVG! Where does this spinner come from? Did you create it or do you have a link to the related license?

Reply:
I was using icon from webpack-svgstore-plugin. I change it now to material sync icon: https://material.io/icons/#ic_sync to not guess about licence.

Reply:
Excellent, thank you.